src/Algebra/Dioid/ShortestDistance.agda

```diff
@@ -26,16 +26,16 @@ data _≡_ : (x y : ShortestDistance) -> Set where
   symmetry     : ∀ {x y   : ShortestDistance} -> x ≡ y ->          y ≡ x
   transitivity : ∀ {x y z : ShortestDistance} -> x ≡ y -> y ≡ z -> x ≡ z
 
-shortest-distance-dioid : Dioid -> ShortestDistance
-shortest-distance-dioid zero = Unreachable
-shortest-distance-dioid one  = Distance Nat.zero
-shortest-distance-dioid (r + s) with shortest-distance-dioid r
-... | Unreachable = shortest-distance-dioid s
-... | Distance n with shortest-distance-dioid s
-... | Unreachable = Distance n
-... | Distance m = Distance (Nat.min n m)
-shortest-distance-dioid (r * s) with shortest-distance-dioid r
-... | Unreachable = Unreachable
-... | Distance n with shortest-distance-dioid s
-... | Unreachable = Unreachable
-... | Distance m = Distance (n Nat.+ m)
+-- shortest-distance-dioid : Dioid -> ShortestDistance
```

Comment:
Shall we delete commented-out code?

Reply:
Actually, I am thinking about removing the module altogether.
Do you think another dioid implementation would have any benefits?